modules/virt-creating-custom-monitoring-label-for-vms.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-custom-monitoring-label-for-vms_{context}"]
+= Creating a custom monitoring label for virtual machines
+
+To enable queries to multiple virtual machines from a single service, create a custom label and assign it to the virtual machine's YAML file.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+* Edit the `template` spec of your virtual machine configuration file. The following example shows the label name `metrics` being added to the label `monitor`.
++
+[source,yaml]
+----
+spec:
+  template:
+    metadata:
+      labels:
+        monitor: metrics
+----
+
```

Comment:
after the template is edited to add the label, the virtual machine must be stopped and then started so a new pod will be created that will have the label.

Reply:
Revamped the content. Please review.